packages/lib-classifier/src/store/annotations/DrawingAnnotation.js

```diff
@@ -0,0 +1,15 @@
+import { types } from 'mobx-state-tree'
+import Annotation from './Annotation'
+import { PointAnnotation } from './DrawingToolAnnotations'
+
+const Drawing = types.model('Drawing', {
+  value: types.array(types.union({
+    dispatcher: (snapshot) => {
+      if (snapshot.toolType === 'point') return PointAnnotation
+    }
```

Comment:
Drawing annotations should record the frame number that the drawing was made on. PFE records this in the `value` object but should it be declared on this model, since it applies to all drawing tools (and adding it to each annotation model would be clunky?)

Reply:
`toolIndex` is another value key that's used by all drawing tools, so could also be declared in one place.

Reply:
Updated to use `Object.assign` to combine the generic attributes `frame` and `toolIndex` with what will be the unique tool attributes, but I'm not sure if there's a more elegant way utilizing MST types. I attempted to use `compose` and `union` unsuccessfully.

Reply:
Instead of trying to merge it together here with `Object.assign`, you can have a generic model declaration in the annotations folder and compose them in the export of `PointAnnotation` similar to other compositions we do elsewhere. Also, `types.frozen` will cause an issue here because the frozen type is intended to be immutable, where this will change as someone is annotating, so `types.model` should be used instead. If free to ask me in person if this isn't making sense.

Reply:
That worked, thanks! Updates incoming...